src/apps/lwaftr/ndp.lua

```diff
@@ -28,6 +28,9 @@ local datagram = require("lib.protocol.datagram")
 local ethernet = require("lib.protocol.ethernet")
 local ipv6     = require("lib.protocol.ipv6")
 
+local alarms = require("lib.yang.alarms")
+alarms.add_to_inventory(lib.require_alarm("apps.lwaftr.ndp"))
```

Comment:
Great.  You know, I apologize for this thought, but I wonder if the .alarm files are worth it: like, the result of `lib.require_alarm('apps.lwaftr.nd')` is going to look like this:
```lua
{
  alarms = {
    alarm_inventory = {
      [{alarm_type_id='ndp-resolution', alarm_type_qualifier=''}]= {
        resource='internal-interface',
        has_clear=true,
        description='Raise up if NDP app cannot resolve IPv6 address'
      }
    }
  }
}
```

Would it make sense to just have this data here literally, and avoid the `.alarm` files?  Then you could just add the inventory elements:

```lua
alarms.add_to_inventory {
  [{alarm_type_id='ndp-resolution', alarm_type_qualifier=''}]= {
    resource='internal-interface',
    has_clear=true,
    description='Raise up if NDP app cannot resolve IPv6 address'
  }
}
```

It would even be possible to make a more normal API that in its implementation creates these alarm objects for you:

```lua
alarms.add_to_inventory('ndp-resolution, '', {
  resource='internal-interface',
  has_clear=true,
  description='Raise up if NDP app cannot resolve IPv6 address'
})
```

Finally I would mention that it's also conceivable that this API could return an object that you could actually use to signal the alarm:

```lua
local resolve_alarm = alarms.declare_alarm('ndp-resolution, '', {
  resource='internal-interface',
  has_clear=true,
  description='Raise up if NDP app cannot resolve IPv6 address'
})

-- ...
resolve_alarm:raise(...)
resolve_alarm:clear(...)
```

Dunno, maybe it is too much.  WDYT?